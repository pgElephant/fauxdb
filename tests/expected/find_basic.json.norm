{
  "cursor": {
    "firstBatch": [
      {
        "_id": "pg_1"
      },
      {
        "_id": "pg_2"
      },
      {
        "_id": "pg_3"
      },
      {
        "_id": "pg_4"
      },
      {
        "_id": "pg_5"
      }
    ],
    "id": 0,
    "ns": "test.collection"
  },
  "ok": 1
}
