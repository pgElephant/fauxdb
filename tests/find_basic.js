db.runCommand({find: "test", limit: 5})
