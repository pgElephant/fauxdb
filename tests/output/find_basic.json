{
  ok: 1,
  cursor: {
    id: Long('0'),
    ns: 'test.collection',
    firstBatch: [
      {
        _id: '1',
        name: 'test_record_1',
        value: 10,
        active: 'f',
        created_at: 2025
      },
      {
        _id: '2',
        name: 'test_record_2',
        value: 20,
        active: 't',
        created_at: 2025
      },
      {
        _id: '3',
        name: 'test_record_3',
        value: 30,
        active: 'f',
        created_at: 2025
      },
      {
        _id: '4',
        name: 'test_record_4',
        value: 40,
        active: 't',
        created_at: 2025
      },
      {
        _id: '5',
        name: 'test_record_5',
        value: 50,
        active: 'f',
        created_at: 2025
      },
      {
        _id: '6',
        name: 'test_record_6',
        value: 60,
        active: 't',
        created_at: 2025
      },
      {
        _id: '7',
        name: 'test_record_7',
        value: 70,
        active: 'f',
        created_at: 2025
      },
      {
        _id: '8',
        name: 'test_record_8',
        value: 80,
        active: 't',
        created_at: 2025
      },
      {
        _id: '9',
        name: 'test_record_9',
        value: 90,
        active: 'f',
        created_at: 2025
      },
      {
        _id: '10',
        name: 'test_record_10',
        value: 100,
        active: 't',
        created_at: 2025
      }
    ]
  }
}
