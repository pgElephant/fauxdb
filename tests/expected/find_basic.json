{
  "ok": 1,
  "cursor": {
    "id": 0,
    "ns": "test.collection",
    "firstBatch": [
      {"_id": "pg_1"},
      {"_id": "pg_2"},
      {"_id": "pg_3"},
      {"_id": "pg_4"},
      {"_id": "pg_5"}
    ]
  }
}
